include/pipettes/core/eeprom.hpp

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include <array>
+#include <concepts>
+#include <cstdint>
+
+#include "common/core/bit_utils.hpp"
+
+/*
+ * A template and helper functions describing
+ */
+
+namespace i2c {
+template <class I2C>
+concept EEPromPolicy =
+    requires(I2C i2c_comms, const uint8_t transmit,
+             std::array<uint8_t, I2C::BUFFER_SIZE>& receive) {
+    {i2c_comms.BUFFER_SIZE};
```

Comment:
Rather than having to dig further for the buffer size parameter, I am making BUFFER_SIZE a member of the EEPromPolicy concept. This allows me to have the line on 30.